src/Microsoft.AspNetCore.Cors/Infrastructure/CorsService.cs

```diff
@@ -133,6 +168,7 @@ public virtual void EvaluatePreflightRequest(HttpContext context, CorsPolicy pol
                 !requestHeaders.All(header => CorsConstants.SimpleRequestHeaders.Contains(header, StringComparer.OrdinalIgnoreCase) ||
```

Comment:
Yeah, that would be nice to know. This code path seems to be only for pre-flight request.